### ISSUE TITLE
Include ansible 2.2 compat

### ISSUE THREAD

**Issue by killua-eu** [Vaizard/mage-sovereign-mailstack#2]
https://github.com/sovereign/sovereign/pull/684

**Comment by ariddell** [comment]
What changed in ansible 2.2?

**Comment by killua-eu** [comment]
@ariddell I had to set up a complete mailstack **really fast** approx 1.5 month ago. I split the project into several files to see exactly what I'm doing + to resolve some problems i had running plain Sovereign. One of them was with encfs, which I think I suspected to be a quotes problem, which might be covered by this pull, but I couldn't explore this and didnt yet find time to do, hence this ticket. I'm wanting to pull this too and test this on a vm to see if this helps to bring in encfs back and/or look into encfs replacements too if I have time (ecryptfs comes into my mind). BTW, I found some small issues with Sovereign that I patched in my fork (and shamefully didn't yet propagate back upstream), see 

- https://github.com/Vaizard/mage.sovereign-mailstack/commits/master
- https://github.com/Vaizard/mage.sovereign-common/commit/0fa7865b8f89a1f3697ac6e18d120ec68f2dd5b4

I'll try to get to that too, unless you are free to cherrypick what seems to make sense for you.
